src/main/java/mate/academy/spring/model/dto/request/OrderRequestDto.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.spring.model.dto.request;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderRequestDto {
+    private List<Long> ticketIds;
+    private LocalDateTime orderDate;
+    private Long userId;
+
+    public List<Long> getTicketIds() {
+        return ticketIds;
+    }
+
+    public void setTicketIds(List<Long> ticketIds) {
```

Comment:
Do you use setters in requestDto? Think about getters in responseDto

Reply:
Done